make-up/code/4840-filip-petrovski/src/app/app.config.ts

```diff
@@ -0,0 +1,8 @@
+import { ApplicationConfig } from '@angular/core';
```

Comment:
I don't know why the application was basically regenerated with a standalone application - but I don't think that it will be able to run at all